docs/tools/piral-inspector.md

```diff
@@ -4,4 +4,215 @@ title: Piral Inspector
+> This option is only available when the user is debugging a Pilet.
+
+#### Visualize component origins
+Activates the option to visualize the origin of the modules as another icon in Piral instance section:
+
+![DebugSettingsVisCompOriginToolPiralInspector](../diagrams/piralinspector_tool_visualizecomponentoriginicon.png)
+
+After clicking on the icon, the origin of each single tile will be displayed:
+
+![DebugSettingsVisCompOriginToolPiralInspector](../diagrams/piralinspector_tool_visualizecomponentorigintile.png)
+
+The origin will also be displayed when hovering over a tile.
+
+
+### Available Pilets
+The Piral Inspector lists all the currently available Pilets. Like illustrated in the table below.
+
+![AvailablePiletsToolPiralInspector](../diagrams/piralinspector_tool_availablepilets.png)
+
+By clicking on the switch button on the left, the user can activate and deactivate each Pilet.
+
+With a click on the "X" on the right, the user can remove each Pilet individually.
+
+### Add Pilets
+There are two options to load additional Pilets into the currently running Piral instance:
+
+#### Add a feed address
+Enter a feed address into the text field and click on the ```"Add"``` button.
+In case you chose a feed you have access to, the corresponding Pilets will be loaded and shown in the tile section and the list of available Pilets. Per default, the new Pilets will be active.
+
+#### Bundle local Pilet and upload it
+First, you need to ```"Browse"``` to the Pilet package file and then ```"Upload"``` it.
+In case you uploaded a proper Pilet, it will be loaded and shown in the tile section and the list of available Pilets. Per default, the new Pilet will be active, but will be given a random name.
+> The Pilet will be loaded into a temporary feed, and this feed (including the Pilets) will be removed when the predefined lifetime (default 60 minutes) expires.
+
+![AddPiletsToolPiralInspector](../diagrams/piralinspector_tool_addpilets.png)
+
+### Registered Routes
+This section shows all currently registered routes, and with a click on a single one, it will call the route.
+
+![RegisteredRoutesToolPiralInspector](../diagrams/piralinspector_tool_registeredroutes.png)
+
+In case the route needs one or more input parameters, a dialog will appear, and the user can provide the required information.
+
+![RegisteredRoutesParameterToolPiralInspector](../diagrams/piralinspector_tool_registeredroutespara.png)
+
+### Events
+All events which occurred since the last refresh of the page are collected and shown in this section.
+
+![EventsToolPiralInspector](../diagrams/piralinspector_tool_events.png)
+
+Besides that, the user can emit a new event:
+
+![EmitEventToolPiralInspector](../diagrams/piralinspector_tool_emitevent.png)
+
+### State Container
+The final section shows the currently available global state (read-only) for the following topics:
+
+![StateContainerToolPiralInspector](../diagrams/piralinspector_tool_statecontainer.png)
+
+#### app
```

Comment:
I don't think the documentation should go into this. After all, everything that is displayed here may be changed (the state container can be fully defined / modified by the Piral instance owner) + other things may appear here (e.g., by some plugin or custom substate).

Reply:
Removed.